api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -35,4 +36,37 @@ message RedisProxy {
 
   // Network settings for the connection pool to the upstream cluster.
   ConnPoolSettings settings = 3 [(validate.rules).message.required = true];
+
+  message RouteMatch {
+    // If specified, the route is a prefix rule that must match the beginning of the keys. The proxy
+    // will always favor the longest matches.
+    string prefix = 1 [(validate.rules).string.min_bytes = 1];
+
+    // Indicates that prefix/path matching should be case insensitive. The default is true.
+    google.protobuf.BoolValue case_sensitive = 2;
+  }
+
+  message RouteAction {
+    string cluster = 1 [(validate.rules).string.min_bytes = 1];
+  }
+
+  message Route {
+    // Name of the route that will be used for instrumentation purposes.
+    string name = 1 [(validate.rules).string.min_bytes = 1];
```

Comment:
is this field used currently?

Reply:
I don't think I've used it yet, but we could potentially use it to push forwarding statistics? I was unsure what I could use it for right now, but noticed that every http routes had names and tried to align as much as I could both APIs.

Reply:
yes, that sounds good.